test/runtime/samples/reactive-values-uninitialised/_config.js

```diff
@@ -0,0 +1,3 @@
+export default {
+	html: `<p>aca</p>`
```

Comment:
looks like you'll need to change this to single quotes to pass the `lint` check

Reply:
oops. 🙈